test/functional/plugin/tohtml_spec.lua

```diff
@@ -162,7 +162,7 @@ describe(':TOhtml', function()
       '<title>title</title>',
       ('<meta name="colorscheme" content="%s"></meta>'):format(api.nvim_get_var('colors_name')),
       '<style>',
-      '* {font-family: dumyfont,monospace}',
+      '* {font-family: "dumyfont",monospace}',
```

Comment:
can we add another font or two on line 152 so that this covers all the logic from this PR?